app/api/dao/user.py

```diff
@@ -9,6 +10,45 @@
 from app.utils.validation_utils import is_email_valid
 
 
+def check_security(password_check):
+    """Returns a boolean if the password is secure"""
+
+    flag = 0
+    flags = {
+        'len': 0,
+        'uppcas': 0,
+        'digits': 0,
+        'specchar': 0,
+        }
+    messages = {
+        'len': messages.INSUFFICIENT_LENGTH,
+        'uppcas': messages.ATLEAST_ONE_UPPERCASE,
+        'digits': messages.ATLEAST_NUMBER,
+        'specchar': messages.ATLEAST_ONE_SPECIAL_CHARACTER,
+        }
+
+    if len(password_check) < 8:
+        flags['len'] += 1
+    else:
+        for ch in password_check:
```

Comment:
lets add lambda function for that maybe `count_chars = lambda str_input, str_chars : sum(str_input.count(c) for c in str_chars)` so then we can call things like `count_chars(test_str, string.digits)` python string has many predefined arrays, please check documentation

Reply:
Okay you want to make a separate function for counting occurrences of a particular character in a string.I will check if there are any predefined ways in python.